app/models/user.rb

```diff
@@ -238,6 +238,20 @@ def unread_issues
     issues.unread_by(self)
   end
 
+  def validate_steamid
+    errors.add :steamid unless
+	  steamid.nil? ||
+      (m = steamid.match(/\A([01]):([01]):(\d{1,10})\Z/)) &&
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.